tests/unit/models/summon.js

```diff
@@ -279,6 +290,17 @@ describe("Summon Model >", function() {
 
       expect(summon.getEndpoint(scope)).toContain("articles/doi/10.1136%2Fbmj.h2575?include=journal");
     });
+
+    it("should build a journal endpoint for an article search result with a journal issn but no article doi", function() {
+      var scope = {
+        document: {
+          content_type: "Journal Article",
+          issns: ["0028-4793"]
+        }
+      };
+
+      expect(summon.getEndpoint(scope)).toContain("search?issns=00284793");
```

Comment:
So glad we've got all these tests to help clarify the requirements and prevent regressions. Great job updating and adding more to these, Tian! 👏 👏

Reply:
`npm test` 236 tests and growing...